spec/tests/angularComponent/ngcOmniboxControllerSpec.js

```diff
@@ -466,6 +466,21 @@ describe('ngcOmnibox.angularComponent.ngcOmniboxController', () => {
         omniboxController.choose('three');
       });
 
+      it('should not add dupes to ngModel when onChosen event prevents/peforms default', (done) => {
+        omniboxController.onChosen = ({$event}) => {
+          $event.preventDefault();
+
+          omniboxController.ngModel.push('three');
+
+          $event.performDefault();
+          expect(omniboxController.ngModel).toEqual(['one', 'two', 'three']);
```

Comment:
I think this would be a little easier for me to understand in code review if this line were duplicated to line 474. The GitHub code review always makes me forget what is happening inside beforeEach() so it's hard to tell what the initial value of `ngModel` is.